Maquetado/scss/navbar.scss

```diff
@@ -1,9 +1,14 @@
+@mixin border-top ($color) {
```

Comment:
I think that not is necessary create a mixin for this

Reply:
Because i use the same in login and navbar